File: pinyougou_parent/pinyougou_manager_web/src/main/webapp/js/service/areasService.js
app.service('areasService', function ($http) {
    //旧的查询方法(弃置)								   
    this.findPage = function (pageNum, pageSize) {
        return $http.get('../areas/findPage.do?pageNum=' + pageNum + '&pageSize=' + pageSize)
    };

    //查询方法,读取列表数据绑定到表单中
    this.search = function (pageNum, pageSize, searchEntity) {
        return $http.post('../areas/findPage.do?pageNum=' + pageNum + "&pageSize=" + pageSize, searchEntity)
    };

    //删除方法
    this.removeAreasList = function (idList) {
        return $http.get('../areas/removeAreas.do?idList=' + idList)
    };

    //根据id查询
    this.findOne = function (id) {
        return $http.get('../areas/findOne.do?id=' + id)
    };

    //修改品牌
    this.updateAreas = function (areas) {
        return $http.post('../areas/updateAreas.do', areas)
    };

    //新增品牌
    this.saveAreas = function (areas) {
        return $http.post('../areas/saveAreas.do', areas)
    };   
});

File: pinyougou_parent/pinyougou_sellergoods_interface/src/main/java/com/pinyougou/sellergoods/service/AreasService.java
package com.pinyougou.sellergoods.service;
import java.util.List;
import com.pinyougou.pojo.TbAreas;							  
import entity.PageResult;
import entity.Result;
/**
 * areas服务层接口
 * @author jieweili
 *
 */
public interface AreasService {

    /**
     * 返回全部列表(已弃置)
     * @return
     */
    List<TbAreas> findAll();
 
    /**
     * 返回分页对象(过时)
     * @param pageNum
     * @param pageSize
     * @return
     */
	 
    PageResult findPage(Integer pageNum, Integer pageSize);

    /**
     * 添加数据
     * @param areas
     * @return
     */
    public Result saveAreas(TbAreas areas);

    /**
     * 根据id查询一个对象
     * @param id
     * @return
     */
    TbAreas getAreasById(Integer id);

    /**
     * 修改对象
     * @param areas
     * @return
     */
    Result updateAreas(TbAreas areas);

    /**
     * 批量删除
     * @param idList
     * @return
     */
    Result removeAreas(Integer[] idList);

    /**
     * 根据条件分页查询
     * @param areas
     * @param pageNum
     * @param pageSize
     * @return
     */
    PageResult listAreasByExample(TbAreas areas, Integer pageNum, Integer pageSize);
 
}
